<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interferometric_Formation</class>
 <widget class="QWidget" name="Interferometric_Formation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>干涉相位生成</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QtWidgetsApplication3.qrc">
    <normaloff>:/QtWidgetsApplication3/bin/icon/InSARapp.ico</normaloff>:/QtWidgetsApplication3/bin/icon/InSARapp.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>目标工程</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>目标节点</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>选取主图像</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_3"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="Isdeflat">
     <property name="text">
      <string>去除平地相位</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Istopo_removal">
     <property name="text">
      <string>去除地形相位</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="iscoherence">
     <property name="text">
      <string>计算相干系数</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="win_w_label">
       <property name="text">
        <string>相关系数估计窗口宽度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="win_w"/>
     </item>
     <item>
      <widget class="QLabel" name="win_h_label">
       <property name="text">
        <string>相关系数估计窗口高度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="win_h"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>多视倍数（距离向）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="multilook_rg"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>多视倍数（方位向）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="multilook_az"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>目标文件夹名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="file_name"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../QtWidgetsApplication3.qrc"/>
 </resources>
 <connections/>
</ui>
